factscore/factscorer.py

```diff
@@ -52,6 +52,8 @@ def save_cache(self):
                 v.save_cache()
         for k, v in self.retrieval.items():
             v.save_cache()
+        if self.af_generator:
+            self.af_generator.save_cache()
```

Comment:
should we call `self.save_cache()` more often? Maybe after every individual generation in the `get_score` function's loop over generations?